refarch-frontend/src/plugins/index.ts

```diff
@@ -0,0 +1,9 @@
+import type { App } from "vue";
+
+import router from "../router";
+import pinia from "./pinia";
+import vuetify from "./vuetify";
```

Comment:
We should always import files using the `@/..` shorthand provided by Vite. This should also be possible in non Vue SFCs.

Reply:
#79